[CI/CD] Night Build is Failing

The night build is failing with the following error:

```java
2021-08-03T22:20:33.0223938Z org.testcontainers.containers.ContainerLaunchException: Timed out waiting for log output matching '.*Server startup complete.*'
2021-08-03T22:20:33.0233454Z 	at org.testcontainers.containers.wait.strategy.LogMessageWaitStrategy.waitUntilReady(LogMessageWaitStrategy.java:49)
2021-08-03T22:20:33.0237705Z 	at org.testcontainers.containers.wait.strategy.AbstractWaitStrategy.waitUntilReady(AbstractWaitStrategy.java:35)
2021-08-03T22:20:33.0255620Z 	at org.testcontainers.containers.GenericContainer.waitUntilContainerStarted(GenericContainer.java:892)
2021-08-03T22:20:33.0258718Z 	at org.testcontainers.containers.GenericContainer.tryStart(GenericContainer.java:440)
2021-08-03T22:20:33.0286130Z 	at org.testcontainers.containers.GenericContainer.lambda$doStart$0(GenericContainer.java:325)
2021-08-03T22:20:33.0289117Z 	at org.rnorth.ducttape.unreliables.Unreliables.retryUntilSuccess(Unreliables.java:81)
2021-08-03T22:20:33.0291617Z 	at org.testcontainers.containers.GenericContainer.doStart(GenericContainer.java:323)
2021-08-03T22:20:33.0293900Z 	at org.testcontainers.containers.GenericContainer.start(GenericContainer.java:311)
2021-08-03T22:20:33.0296349Z 	at org.eclipse.dirigible.api.rabbitmq.RabbitMQFacadeTest.setUp(RabbitMQFacadeTest.java:34)
2021-08-03T22:20:33.0300764Z 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
2021-08-03T22:20:33.0303518Z 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
2021-08-03T22:20:33.0306134Z 	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
2021-08-03T22:20:33.0308054Z 	at java.base/java.lang.reflect.Method.invoke(Method.java:567)
2021-08-03T22:20:33.0309444Z 	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
2021-08-03T22:20:33.0311406Z 	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
2021-08-03T22:20:33.0422431Z 	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
2021-08-03T22:20:33.0424806Z 	at org.junit.internal.runners.statements.RunBefores.invokeMethod(RunBefores.java:33)
2021-08-03T22:20:33.0427058Z 	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:24)
2021-08-03T22:20:33.0428759Z 	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
2021-08-03T22:20:33.0430247Z 	at org.junit.runners.BlockJUnit4ClassRunner$1.evaluate(BlockJUnit4ClassRunner.java:100)
2021-08-03T22:20:33.0431832Z 	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:366)
2021-08-03T22:20:33.0433537Z 	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:103)
2021-08-03T22:20:33.0435601Z 	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:63)
2021-08-03T22:20:33.0437629Z 	at org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
2021-08-03T22:20:33.0438831Z 	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)
2021-08-03T22:20:33.0440204Z 	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)
2021-08-03T22:20:33.0441628Z 	at org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)
2021-08-03T22:20:33.0442903Z 	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)
2021-08-03T22:20:33.0444373Z 	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
2021-08-03T22:20:33.0445595Z 	at org.junit.runners.ParentRunner.run(ParentRunner.java:413)
2021-08-03T22:20:33.0447283Z 	at org.apache.maven.surefire.junit4.JUnit4Provider.execute(JUnit4Provider.java:365)
2021-08-03T22:20:33.0449646Z 	at org.apache.maven.surefire.junit4.JUnit4Provider.executeWithRerun(JUnit4Provider.java:273)
2021-08-03T22:20:33.0452729Z 	at org.apache.maven.surefire.junit4.JUnit4Provider.executeTestSet(JUnit4Provider.java:238)
2021-08-03T22:20:33.0455020Z 	at org.apache.maven.surefire.junit4.JUnit4Provider.invoke(JUnit4Provider.java:159)
2021-08-03T22:20:33.0457797Z 	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:384)
2021-08-03T22:20:33.0460991Z 	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:345)
2021-08-03T22:20:33.0463327Z 	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:126)
2021-08-03T22:20:33.0465271Z 	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:418)
2021-08-03T22:20:33.0469072Z 22:20:33.019 [main] DEBUG org.testcontainers.shaded.com.github.dockerjava.core.command.AbstrDockerCmd - Cmd: ad79f6605fc72c98aa633a41276c7d2df5295d858fae12a21279fa92a3c3b2ce,false
2021-08-03T22:20:33.0474273Z 22:20:33.019 [main] DEBUG org.testcontainers.shaded.com.github.dockerjava.core.exec.InspectContainerCmdExec - GET: DefaultWebTarget{path=[/containers/ad79f6605fc72c98aa633a41276c7d2df5295d858fae12a21279fa92a3c3b2ce/json], queryParams={}}
2021-08-03T22:20:33.0477720Z 22:20:33.023 [main] ERROR 🐳 [rabbitmq:alpine] - Could not start container
2021-08-03T22:20:33.0478668Z java.lang.IllegalStateException: Container exited with code 1
2021-08-03T22:20:33.0480343Z 	at org.testcontainers.containers.GenericContainer.tryStart(GenericContainer.java:469)
2021-08-03T22:20:33.0482584Z 	at org.testcontainers.containers.GenericContainer.lambda$doStart$0(GenericContainer.java:325)
2021-08-03T22:20:33.0484906Z 	at org.rnorth.ducttape.unreliables.Unreliables.retryUntilSuccess(Unreliables.java:81)
2021-08-03T22:20:33.0487228Z 	at org.testcontainers.containers.GenericContainer.doStart(GenericContainer.java:323)
2021-08-03T22:20:33.0489380Z 	at org.testcontainers.containers.GenericContainer.start(GenericContainer.java:311)
2021-08-03T22:20:33.0491679Z 	at org.eclipse.dirigible.api.rabbitmq.RabbitMQFacadeTest.setUp(RabbitMQFacadeTest.java:34)
2021-08-03T22:20:33.0493952Z 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
2021-08-03T22:20:33.0496171Z 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
2021-08-03T22:20:33.0498767Z 	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
2021-08-03T22:20:33.0500875Z 	at java.base/java.lang.reflect.Method.invoke(Method.java:567)
2021-08-03T22:20:33.0502278Z 	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
2021-08-03T22:20:33.0504252Z 	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
2021-08-03T22:20:33.0506493Z 	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
2021-08-03T22:20:33.0508820Z 	at org.junit.internal.runners.statements.RunBefores.invokeMethod(RunBefores.java:33)
2021-08-03T22:20:33.0511063Z 	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:24)
2021-08-03T22:20:33.0512737Z 	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
2021-08-03T22:20:33.0514267Z 	at org.junit.runners.BlockJUnit4ClassRunner$1.evaluate(BlockJUnit4ClassRunner.java:100)
2021-08-03T22:20:33.0515852Z 	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:366)
2021-08-03T22:20:33.0517573Z 	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:103)
2021-08-03T22:20:33.0538617Z 	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:63)
2021-08-03T22:20:33.0627549Z 	at org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
2021-08-03T22:20:33.0628798Z 	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)
2021-08-03T22:20:33.0630179Z 	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)
2021-08-03T22:20:33.0631604Z 	at org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)
2021-08-03T22:20:33.0632887Z 	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)
2021-08-03T22:20:33.0634093Z 	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
2021-08-03T22:20:33.0635318Z 	at org.junit.runners.ParentRunner.run(ParentRunner.java:413)
2021-08-03T22:20:33.0637005Z 	at org.apache.maven.surefire.junit4.JUnit4Provider.execute(JUnit4Provider.java:365)
2021-08-03T22:20:33.0639367Z 	at org.apache.maven.surefire.junit4.JUnit4Provider.executeWithRerun(JUnit4Provider.java:273)
2021-08-03T22:20:33.0641952Z 	at org.apache.maven.surefire.junit4.JUnit4Provider.executeTestSet(JUnit4Provider.java:238)
2021-08-03T22:20:33.0644245Z 	at org.apache.maven.surefire.junit4.JUnit4Provider.invoke(JUnit4Provider.java:159)
2021-08-03T22:20:33.0647006Z 	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:384)
2021-08-03T22:20:33.0650062Z 	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:345)
2021-08-03T22:20:33.0652364Z 	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:126)
2021-08-03T22:20:33.0654305Z 	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:418)
2021-08-03T22:20:33.0691648Z 22:20:33.066 [main] ERROR 🐳 [rabbitmq:alpine] - Log output from the failed container:
2021-08-03T22:20:33.0692797Z error: RABBITMQ_DEFAULT_PASS is set but deprecated
2021-08-03T22:20:33.0693713Z error: deprecated environment variables detected
2021-08-03T22:20:33.0694493Z 
2021-08-03T22:20:33.0695560Z Please use a configuration file instead; visit https://www.rabbitmq.com/configure.html to learn more
2021-08-03T22:20:33.0696453Z 
2021-08-03T22:20:33.0696808Z 
2021-08-03T22:20:33.0900943Z [ERROR] Tests run: 2, Failures: 0, Errors: 2, Skipped: 0, Time elapsed: 129.716 s <<< FAILURE! - in org.eclipse.dirigible.api.rabbitmq.RabbitMQFacadeTest
2021-08-03T22:20:33.0906849Z [ERROR] send(org.eclipse.dirigible.api.rabbitmq.RabbitMQFacadeTest)  Time elapsed: 69.074 s  <<< ERROR!
2021-08-03T22:20:33.0911609Z org.testcontainers.containers.ContainerLaunchException: Container startup failed
2021-08-03T22:20:33.0916793Z 	at org.eclipse.dirigible.api.rabbitmq.RabbitMQFacadeTest.setUp(RabbitMQFacadeTest.java:34)
2021-08-03T22:20:33.0927988Z Caused by: org.rnorth.ducttape.RetryCountExceededException: Retry limit hit with exception
2021-08-03T22:20:33.0930533Z 	at org.eclipse.dirigible.api.rabbitmq.RabbitMQFacadeTest.setUp(RabbitMQFacadeTest.java:34)
2021-08-03T22:20:33.0933133Z Caused by: org.testcontainers.containers.ContainerLaunchException: Could not create/start container
2021-08-03T22:20:33.0936168Z 	at org.eclipse.dirigible.api.rabbitmq.RabbitMQFacadeTest.setUp(RabbitMQFacadeTest.java:34)
2021-08-03T22:20:33.0938290Z Caused by: java.lang.IllegalStateException: Container exited with code 1
2021-08-03T22:20:33.0940602Z 	at org.eclipse.dirigible.api.rabbitmq.RabbitMQFacadeTest.setUp(RabbitMQFacadeTest.java:34)
2021-08-03T22:20:33.0942793Z 
2021-08-03T22:20:33.0945294Z [ERROR] rabbitMQIntegration(org.eclipse.dirigible.api.rabbitmq.RabbitMQFacadeTest)  Time elapsed: 60.57 s  <<< ERROR!
2021-08-03T22:20:33.0947490Z org.testcontainers.containers.ContainerLaunchException: Container startup failed
2021-08-03T22:20:33.0953507Z 	at org.eclipse.dirigible.api.rabbitmq.RabbitMQFacadeTest.setUp(RabbitMQFacadeTest.java:34)
2021-08-03T22:20:33.0964306Z Caused by: org.rnorth.ducttape.RetryCountExceededException: Retry limit hit with exception
2021-08-03T22:20:33.0972740Z 	at org.eclipse.dirigible.api.rabbitmq.RabbitMQFacadeTest.setUp(RabbitMQFacadeTest.java:34)
2021-08-03T22:20:33.0975106Z Caused by: org.testcontainers.containers.ContainerLaunchException: Could not create/start container
2021-08-03T22:20:33.0977656Z 	at org.eclipse.dirigible.api.rabbitmq.RabbitMQFacadeTest.setUp(RabbitMQFacadeTest.java:34)
2021-08-03T22:20:33.0979508Z Caused by: java.lang.IllegalStateException: Container exited with code 1
2021-08-03T22:20:33.0981585Z 	at org.eclipse.dirigible.api.rabbitmq.RabbitMQFacadeTest.setUp(RabbitMQFacadeTest.java:34)
2021-08-03T22:20:33.0982888Z 
2021-08-03T22:20:33.4173602Z [INFO] 
2021-08-03T22:20:33.4174654Z [INFO] Results:
2021-08-03T22:20:33.4175247Z [INFO] 
2021-08-03T22:20:33.4175783Z [ERROR] Errors: 
2021-08-03T22:20:33.4177422Z [ERROR]   RabbitMQFacadeTest.setUp:34 » ContainerLaunch Container startup failed
2021-08-03T22:20:33.4181815Z [ERROR]   RabbitMQFacadeTest.setUp:34 » ContainerLaunch Container startup failed
2021-08-03T22:20:33.4182808Z [INFO] 
2021-08-03T22:20:33.4183552Z [ERROR] Tests run: 2, Failures: 0, Errors: 2, Skipped: 0
2021-08-03T22:20:33.4184190Z [INFO] 
2021-08-03T22:20:33.4219088Z [INFO] ------------------------------------------------------------------------
2021-08-03T22:20:33.4230711Z [INFO] Reactor Summary for Dirigible - Parent 6.0.0-SNAPSHOT:
2021-08-03T22:20:33.4231571Z [INFO] 
2021-08-03T22:20:33.4241191Z [INFO] Dirigible - Parent ................................. SUCCESS [01:00 min]
2021-08-03T22:20:33.4242691Z [INFO] Commons - Parent ................................... SUCCESS [  0.037 s]
2021-08-03T22:20:33.4244001Z [INFO] Commons - Configurations ........................... SUCCESS [ 19.416 s]
2021-08-03T22:20:33.4245146Z [INFO] Commons - API ...................................... SUCCESS [  7.810 s]
2021-08-03T22:20:33.4246526Z [INFO] Database - Parent .................................. SUCCESS [  0.085 s]
2021-08-03T22:20:33.4247636Z [INFO] Database - SQL Builders ............................ SUCCESS [ 33.217 s]
2021-08-03T22:20:33.4248766Z [INFO] Database - Persistence ............................. SUCCESS [ 12.623 s]
2021-08-03T22:20:33.4249858Z [INFO] Commons - Timeout .................................. SUCCESS [  2.367 s]
2021-08-03T22:20:33.4250978Z [INFO] Commons - Health ................................... SUCCESS [  2.701 s]
2021-08-03T22:20:33.4252114Z [INFO] Database - API ..................................... SUCCESS [  7.586 s]
2021-08-03T22:20:33.4253436Z [INFO] Repository - Parent ................................ SUCCESS [  0.029 s]
2021-08-03T22:20:33.4254681Z [INFO] Repository - API ................................... SUCCESS [  6.456 s]
2021-08-03T22:20:33.4256871Z [INFO] API - Parent ....................................... SUCCESS [  0.064 s]
2021-08-03T22:20:33.4258174Z [INFO] API - Facade - Parent .............................. SUCCESS [  0.079 s]
2021-08-03T22:20:33.4259276Z [INFO] API - Facade - HTTP ................................ SUCCESS [  4.831 s]
2021-08-03T22:20:33.4260724Z [INFO] API - Facade - Security ............................ SUCCESS [  2.642 s]
2021-08-03T22:20:33.4261917Z [INFO] Commons - XML to JSON .............................. SUCCESS [  3.188 s]
2021-08-03T22:20:33.4263009Z [INFO] Commons - Test ..................................... SUCCESS [  2.482 s]
2021-08-03T22:20:33.4264078Z [INFO] Database - Derby ................................... SUCCESS [  5.460 s]
2021-08-03T22:20:33.4265145Z [INFO] Repository - Search ................................ SUCCESS [  3.101 s]
2021-08-03T22:20:33.4266262Z [INFO] Repository - ZIP ................................... SUCCESS [  2.519 s]
2021-08-03T22:20:33.4267348Z [INFO] Service - Parent ................................... SUCCESS [  0.067 s]
2021-08-03T22:20:33.4268429Z [INFO] Service - Anonymous ................................ SUCCESS [  2.558 s]
2021-08-03T22:20:33.4269555Z [INFO] Repository - API - Test ............................ SUCCESS [  4.678 s]
2021-08-03T22:20:33.4270662Z [INFO] Repository - Local ................................. SUCCESS [  8.665 s]
2021-08-03T22:20:33.4271746Z [INFO] Database - Data Models ............................. SUCCESS [ 11.270 s]
2021-08-03T22:20:33.4272850Z [INFO] Engine - Parent .................................... SUCCESS [  0.021 s]
2021-08-03T22:20:33.4274185Z [INFO] Engine - API ....................................... SUCCESS [  3.897 s]
2021-08-03T22:20:33.4276095Z [INFO] Repository - Master ................................ SUCCESS [  3.540 s]
2021-08-03T22:20:33.4277016Z [INFO] Core - Parent ...................................... SUCCESS [  0.022 s]
2021-08-03T22:20:33.4278519Z [INFO] Core - Scheduler ................................... SUCCESS [ 11.150 s]
2021-08-03T22:20:33.4280972Z [INFO] Core - Extensions .................................. SUCCESS [  7.078 s]
2021-08-03T22:20:33.4283802Z [INFO] Commons - Process .................................. SUCCESS [  3.285 s]
2021-08-03T22:20:33.4284669Z [INFO] Engine - Command ................................... SUCCESS [  4.327 s]
2021-08-03T22:20:33.4285830Z [INFO] API - Facade - Core ................................ SUCCESS [  9.364 s]
2021-08-03T22:20:33.4286884Z [INFO] Database - Dynamic ................................. SUCCESS [  2.710 s]
2021-08-03T22:20:33.4289124Z [INFO] API - Facade - Database ............................ SUCCESS [  2.689 s]
2021-08-03T22:20:33.4289986Z [INFO] Engine - JavaScript ................................ SUCCESS [  8.545 s]
2021-08-03T22:20:33.4292027Z [INFO] API - Facade - HTTP Async .......................... SUCCESS [  3.114 s]
2021-08-03T22:20:33.4292884Z [INFO] Core - Indexing .................................... SUCCESS [  4.385 s]
2021-08-03T22:20:33.4293694Z [INFO] API - Facade - Indexing ............................ SUCCESS [  2.353 s]
2021-08-03T22:20:33.4296469Z [INFO] API - Facade - IO .................................. SUCCESS [  5.180 s]
2021-08-03T22:20:33.4297287Z [INFO] API - Facade - Log ................................. SUCCESS [  2.715 s]
2021-08-03T22:20:33.4298065Z [INFO] API - Facade - Mail ................................ SUCCESS [  2.828 s]
2021-08-03T22:20:33.4301072Z [INFO] API - Facade - Utils ............................... SUCCESS [  3.521 s]
2021-08-03T22:20:33.4302718Z [INFO] CMS - Parent ....................................... SUCCESS [  0.017 s]
2021-08-03T22:20:33.4303512Z [INFO] CMS - API .......................................... SUCCESS [  2.349 s]
2021-08-03T22:20:33.4305899Z [INFO] Core - Security .................................... SUCCESS [  9.013 s]
2021-08-03T22:20:33.4306710Z [INFO] API - Facade - CMS ................................. SUCCESS [  2.672 s]
2021-08-03T22:20:33.4307497Z [INFO] Ext - Parent ....................................... SUCCESS [  0.023 s]
2021-08-03T22:20:33.4308293Z [INFO] Ext - Modules - Parent ............................. SUCCESS [  0.019 s]
2021-08-03T22:20:33.4311787Z [INFO] API - Facade - Spark ............................... SUCCESS [ 18.406 s]
2021-08-03T22:20:33.4312684Z [INFO] Ext - Modules - Parent ............................. SUCCESS [  0.020 s]
2021-08-03T22:20:33.4313482Z [INFO] Ext - Spark ........................................ SUCCESS [  0.384 s]
2021-08-03T22:20:33.4314241Z [INFO] CMS - Internal ..................................... SUCCESS [  5.853 s]
2021-08-03T22:20:33.4318818Z [INFO] Database - H2 ...................................... SUCCESS [  4.304 s]
2021-08-03T22:20:33.4321948Z [INFO] Core - Publisher ................................... SUCCESS [  6.488 s]
2021-08-03T22:20:33.4322824Z [INFO] Service - Repository ............................... SUCCESS [  3.556 s]
2021-08-03T22:20:33.4323658Z [INFO] Core - Workspace ................................... SUCCESS [  7.168 s]
2021-08-03T22:20:33.4324506Z [INFO] Core - Generation .................................. SUCCESS [  3.317 s]
2021-08-03T22:20:33.4325363Z [INFO] API - Facade - Platform ............................ SUCCESS [  3.550 s]
2021-08-03T22:20:33.4326205Z [INFO] API - Facade - RabbitMQ ............................ FAILURE [02:13 min]
2021-08-03T22:20:33.4327406Z [INFO] API - Facade - Elasticsearch ....................... SKIPPED
2021-08-03T22:20:33.4328463Z [INFO] API - Facade - Redis ............................... SKIPPED
2021-08-03T22:20:33.4329690Z [INFO] API - Facade - Etcd ................................ SKIPPED
2021-08-03T22:20:33.4330733Z [INFO] API - JavaScript - Parent .......................... SKIPPED
2021-08-03T22:20:33.4331798Z [INFO] API - JavaScript - Security ........................ SKIPPED
2021-08-03T22:20:33.4334199Z [INFO] API - JavaScript - Core ............................ SKIPPED
2021-08-03T22:20:33.4337349Z [INFO] API - JavaScript - HTTP ............................ SKIPPED
2021-08-03T22:20:33.4338183Z [INFO] API - JavaScript - Utils ........................... SKIPPED
2021-08-03T22:20:33.4339038Z [INFO] API - JavaScript - Database ........................ SKIPPED
2021-08-03T22:20:33.4342822Z [INFO] API - JavaScript - IO .............................. SKIPPED
2021-08-03T22:20:33.4343897Z [INFO] API - JavaScript - Indexing ........................ SKIPPED
2021-08-03T22:20:33.4344812Z [INFO] API - JavaScript - Log ............................. SKIPPED
2021-08-03T22:20:33.4345687Z [INFO] Resources - CMS - Parent ........................... SKIPPED
2021-08-03T22:20:33.4346588Z [INFO] API - JavaScript - CMS ............................. SKIPPED
2021-08-03T22:20:33.4347809Z [INFO] API - JavaScript - Platform ........................ SKIPPED
2021-08-03T22:20:33.4349026Z [INFO] Ext - Redis ........................................ SKIPPED
2021-08-03T22:20:33.4349823Z [INFO] Ext - Etcd ......................................... SKIPPED
2021-08-03T22:20:33.4350605Z [INFO] Ext - Rabbitmq ..................................... SKIPPED
2021-08-03T22:20:33.4351485Z [INFO] API - JavaScript - Messaging ....................... SKIPPED
2021-08-03T22:20:33.4355572Z [INFO] Ext - Elasticsearch ................................ SKIPPED
2021-08-03T22:20:33.4357070Z [INFO] API - Facade - Tests ............................... SKIPPED
2021-08-03T22:20:33.4357900Z [INFO] Engine - JavaScript - GraalVM ...................... SKIPPED
2021-08-03T22:20:33.4381954Z [INFO] Core - Messaging ................................... SKIPPED
2021-08-03T22:20:33.4387737Z [INFO] API - Facade - Messaging ........................... SKIPPED
2021-08-03T22:20:33.4393467Z [INFO] API - Facade - Messaging ........................... SKIPPED
2021-08-03T22:20:33.4399127Z [INFO] API - Facade - All ................................. SKIPPED
2021-08-03T22:20:33.4404925Z [INFO] API - JavaScript - Documents ....................... SKIPPED
2021-08-03T22:20:33.4411189Z [INFO] API - JavaScript - Mail ............................ SKIPPED
2021-08-03T22:20:33.4417554Z [INFO] API - JavaScript - Test ............................ SKIPPED
2021-08-03T22:20:33.4423257Z [INFO] API - JavaScript - Net ............................. SKIPPED
2021-08-03T22:20:33.4428573Z [INFO] API - JavaScript - All ............................. SKIPPED
2021-08-03T22:20:33.4433935Z [INFO] API ................................................ SKIPPED
2021-08-03T22:20:33.4439314Z [INFO] IDE - Parent ....................................... SKIPPED
2021-08-03T22:20:33.4444557Z [INFO] IDE - UI - Parent .................................. SKIPPED
2021-08-03T22:20:33.4449757Z [INFO] IDE - UI - Branding ................................ SKIPPED
2021-08-03T22:20:33.4454965Z [INFO] IDE - UI - Core .................................... SKIPPED
2021-08-03T22:20:33.4460377Z [INFO] IDE - UI - Workbench ............................... SKIPPED
2021-08-03T22:20:33.4465685Z [INFO] IDE - UI - Terminal ................................ SKIPPED
2021-08-03T22:20:33.4471127Z [INFO] IDE - UI - Console ................................. SKIPPED
2021-08-03T22:20:33.4476308Z [INFO] IDE - UI - Swagger ................................. SKIPPED
2021-08-03T22:20:33.4482088Z [INFO] IDE - UI - Workspace ............................... SKIPPED
2021-08-03T22:20:33.4488954Z [INFO] IDE - UI - Preview ................................. SKIPPED
2021-08-03T22:20:33.4494796Z [INFO] IDE - UI - Database ................................ SKIPPED
2021-08-03T22:20:33.4501372Z [INFO] IDE - UI - Git ..................................... SKIPPED
2021-08-03T22:20:33.4502778Z [INFO] IDE - UI - Repository .............................. SKIPPED
2021-08-03T22:20:33.4504235Z [INFO] IDE - UI - Schema .................................. SKIPPED
2021-08-03T22:20:33.4505385Z [INFO] IDE - UI - Entity Data Modeler ..................... SKIPPED
2021-08-03T22:20:33.4506505Z [INFO] IDE - UI - Monaco .................................. SKIPPED
2021-08-03T22:20:33.4507726Z [INFO] IDE - UI - Monaco - Extensions ..................... SKIPPED
2021-08-03T22:20:33.4508858Z [INFO] IDE - UI - Debugger ................................ SKIPPED
2021-08-03T22:20:33.4510079Z [INFO] IDE - UI - Logs .................................... SKIPPED
2021-08-03T22:20:33.4512192Z [INFO] IDE - UI - Jobs .................................... SKIPPED
2021-08-03T22:20:33.4514308Z [INFO] IDE - UI - Extensions .............................. SKIPPED
2021-08-03T22:20:33.4554131Z [INFO] IDE - UI - Operations .............................. SKIPPED
2021-08-03T22:20:33.4555377Z [INFO] IDE - UI - Security ................................ SKIPPED
2021-08-03T22:20:33.4556532Z [INFO] IDE - UI - Listeners ............................... SKIPPED
2021-08-03T22:20:33.4558436Z [INFO] IDE - UI - Data Structures ......................... SKIPPED
2021-08-03T22:20:33.4560128Z [INFO] IDE - UI - Registry ................................ SKIPPED
2021-08-03T22:20:33.4560910Z [INFO] IDE - UI - Plugins ................................. SKIPPED
2021-08-03T22:20:33.4561693Z [INFO] IDE - UI - Discussions ............................. SKIPPED
2021-08-03T22:20:33.4562483Z [INFO] IDE - UI - Deploy Manager .......................... SKIPPED
2021-08-03T22:20:33.4563558Z [INFO] IDE - UI - Websockets .............................. SKIPPED
2021-08-03T22:20:33.4564328Z [INFO] IDE - UI - Form Builder ............................ SKIPPED
2021-08-03T22:20:33.4565067Z [INFO] IDE - UI ........................................... SKIPPED
2021-08-03T22:20:33.4565800Z [INFO] IDE - Service - Parent ............................. SKIPPED
2021-08-03T22:20:33.4566605Z [INFO] IDE - Service - Console ............................ SKIPPED
2021-08-03T22:20:33.4567403Z [INFO] IDE - Service - Core ............................... SKIPPED
2021-08-03T22:20:33.4568189Z [INFO] IDE - Service - Databases .......................... SKIPPED
2021-08-03T22:20:33.4568978Z [INFO] Service - Core ..................................... SKIPPED
2021-08-03T22:20:33.4569752Z [INFO] IDE - Service - Workspaces ......................... SKIPPED
2021-08-03T22:20:33.4570592Z [INFO] IDE - Service - Generation ......................... SKIPPED
2021-08-03T22:20:33.4571359Z [INFO] Core - Git ......................................... SKIPPED
2021-08-03T22:20:33.4572102Z [INFO] IDE - Service - Git ................................ SKIPPED
2021-08-03T22:20:33.4572899Z [INFO] IDE - Service - Publisher .......................... SKIPPED
2021-08-03T22:20:33.4573703Z [INFO] IDE - Service - Terminal ........................... SKIPPED
2021-08-03T22:20:33.4574483Z [INFO] IDE - Services ..................................... SKIPPED
2021-08-03T22:20:33.4575029Z [INFO] IDE ................................................ SKIPPED
2021-08-03T22:20:33.4575787Z [INFO] IDE - UI Group - JavaScript ........................ SKIPPED
2021-08-03T22:20:33.4576565Z [INFO] IDE - Services ..................................... SKIPPED
2021-08-03T22:20:33.4577329Z [INFO] IDE - JavaScript ................................... SKIPPED
2021-08-03T22:20:33.4578120Z [INFO] IDE - UI Group - Database .......................... SKIPPED
2021-08-03T22:20:33.4578881Z [INFO] IDE - Services ..................................... SKIPPED
2021-08-03T22:20:33.4579640Z [INFO] IDE - Database ..................................... SKIPPED
2021-08-03T22:20:33.4581020Z [INFO] Template - Parent .................................. SKIPPED
2021-08-03T22:20:33.4584805Z [INFO] Template - Hello World ............................. SKIPPED
2021-08-03T22:20:33.4589536Z [INFO] Template - HTTP Client ............................. SKIPPED
2021-08-03T22:20:33.4597356Z [INFO] Template - Database Access ......................... SKIPPED
2021-08-03T22:20:33.4602467Z [INFO] Template - Database Table .......................... SKIPPED
2021-08-03T22:20:33.4607240Z [INFO] Template - Database View ........................... SKIPPED
2021-08-03T22:20:33.4611965Z [INFO] Template - HTML .................................... SKIPPED
2021-08-03T22:20:33.4618625Z [INFO] Template - Bookstore ............................... SKIPPED
2021-08-03T22:20:33.4623603Z [INFO] Template - Job ..................................... SKIPPED
2021-08-03T22:20:33.4628226Z [INFO] Template - Listener ................................ SKIPPED
2021-08-03T22:20:33.4632930Z [INFO] Template - Extension View .......................... SKIPPED
2021-08-03T22:20:33.4641232Z [INFO] Template - Extension Perspective ................... SKIPPED
2021-08-03T22:20:33.4642363Z [INFO] Template - Application Launchpad (Angular) ......... SKIPPED
2021-08-03T22:20:33.4643303Z [INFO] Template - Application - Schema .................... SKIPPED
2021-08-03T22:20:33.4652650Z [INFO] Template - Application - DAO ....................... SKIPPED
2021-08-03T22:20:33.4653539Z [INFO] Template - Application - Data ...................... SKIPPED
2021-08-03T22:20:33.4654403Z [INFO] Template - Application - REST ...................... SKIPPED
2021-08-03T22:20:33.4655280Z [INFO] Template - Application - Feed ...................... SKIPPED
2021-08-03T22:20:33.4656169Z [INFO] Template - Application - UI - AngularJS ............ SKIPPED
2021-08-03T22:20:33.4657122Z [INFO] Template - Full-stack Application (Angular) ........ SKIPPED
2021-08-03T22:20:33.4658266Z [INFO] Template - Mobile - Hello World .................... SKIPPED
2021-08-03T22:20:33.4659159Z [INFO] Template - AngularJS Form Generator ................ SKIPPED
2021-08-03T22:20:33.4660054Z [INFO] Template - Application - OData ..................... SKIPPED
2021-08-03T22:20:33.4661126Z [INFO] Template - OpenUI5 Wizard .......................... SKIPPED
2021-08-03T22:20:33.4661990Z [INFO] Template - OpenUI5 Simple Form ..................... SKIPPED
2021-08-03T22:20:33.4662836Z [INFO] Template - OpenUI5 Tiles ........................... SKIPPED
2021-08-03T22:20:33.4663715Z [INFO] Template - OpenUI5 Planning Calendar ............... SKIPPED
2021-08-03T22:20:33.4664574Z [INFO] Template - OpenUI5 List ............................ SKIPPED
2021-08-03T22:20:33.4665425Z [INFO] Template - OpenUI5 Master Detail ................... SKIPPED
2021-08-03T22:20:33.4666283Z [INFO] Template - OpenUI5 Table ........................... SKIPPED
2021-08-03T22:20:33.4667097Z [INFO] Template - Websocket ............................... SKIPPED
2021-08-03T22:20:33.4667971Z [INFO] Template - Application - DAO - MongoDB ............. SKIPPED
2021-08-03T22:20:33.4668874Z [INFO] Template - Application - REST - MongoDB ............ SKIPPED
2021-08-03T22:20:33.4669828Z [INFO] Template - Application - UI - AngularJS - MongoDB .. SKIPPED
2021-08-03T22:20:33.4670516Z [INFO] Templates .......................................... SKIPPED
2021-08-03T22:20:33.4671397Z [INFO] Template - Full-stack Application (OpenUI5) ........ SKIPPED
2021-08-03T22:20:33.4672291Z [INFO] Resources - Parent ................................. SKIPPED
2021-08-03T22:20:33.4673068Z [INFO] Resources - Fonts .................................. SKIPPED
2021-08-03T22:20:33.4673896Z [INFO] Resources - GoldenLayout ........................... SKIPPED
2021-08-03T22:20:33.4674711Z [INFO] Resources - Dev Tools .............................. SKIPPED
2021-08-03T22:20:33.4675527Z [INFO] Resources - Resources .............................. SKIPPED
2021-08-03T22:20:33.4676333Z [INFO] Resources - Themes ................................. SKIPPED
2021-08-03T22:20:33.4677148Z [INFO] Resources - Theme - Default ........................ SKIPPED
2021-08-03T22:20:33.4677978Z [INFO] Resources - Theme - Fiori .......................... SKIPPED
2021-08-03T22:20:33.4678576Z [INFO] Resources .......................................... SKIPPED
2021-08-03T22:20:33.4679302Z [INFO] Ext - Acorn ........................................ SKIPPED
2021-08-03T22:20:33.4680210Z [INFO] Ext - Jasmine ...................................... SKIPPED
2021-08-03T22:20:33.4680941Z [INFO] Ext - QUnit ........................................ SKIPPED
2021-08-03T22:20:33.4681644Z [INFO] Ext - Kafka ........................................ SKIPPED
2021-08-03T22:20:33.4682370Z [INFO] Ext - MongoDB ...................................... SKIPPED
2021-08-03T22:20:33.4683105Z [INFO] Ext - Cassandra .................................... SKIPPED
2021-08-03T22:20:33.4683870Z [INFO] API - Facade - Kafka ............................... SKIPPED
2021-08-03T22:20:33.4764817Z [INFO] API - Facade - Kafka ............................... SKIPPED
2021-08-03T22:20:33.4765756Z [INFO] API - Facade - Cassandra ........................... SKIPPED
2021-08-03T22:20:33.4766346Z [INFO] Ext ................................................ SKIPPED
2021-08-03T22:20:33.4807614Z [INFO] Engine - Job ....................................... SKIPPED
2021-08-03T22:20:33.4808666Z [INFO] Engine - Listener .................................. SKIPPED
2021-08-03T22:20:33.4809496Z [INFO] Engine - Web ....................................... SKIPPED
2021-08-03T22:20:33.4810233Z [INFO] Engine - Wiki ...................................... SKIPPED
2021-08-03T22:20:33.4810986Z [INFO] OData - Parent ..................................... SKIPPED
2021-08-03T22:20:33.4812979Z [INFO] OData - API ........................................ SKIPPED
2021-08-03T22:20:33.4813875Z [INFO] OData - Core ....................................... SKIPPED
2021-08-03T22:20:33.4814913Z [INFO] Engine - OData ..................................... SKIPPED
2021-08-03T22:20:33.4815469Z [INFO] Engine ............................................. SKIPPED
2021-08-03T22:20:33.4816298Z [INFO] Service - Extensions ............................... SKIPPED
2021-08-03T22:20:33.4817097Z [INFO] Service - Registry ................................. SKIPPED
2021-08-03T22:20:33.4817892Z [INFO] Service - Security ................................. SKIPPED
2021-08-03T22:20:33.4818670Z [INFO] Service - Theme .................................... SKIPPED
2021-08-03T22:20:33.4819459Z [INFO] Service - Transport ................................ SKIPPED
2021-08-03T22:20:33.4820599Z [INFO] Database - Data Structures ......................... SKIPPED
2021-08-03T22:20:33.4821490Z [INFO] Core - Websockets .................................. SKIPPED
2021-08-03T22:20:33.4822302Z [INFO] Service - Operations ............................... SKIPPED
2021-08-03T22:20:33.4823862Z [INFO] Service - Websockets ............................... SKIPPED
2021-08-03T22:20:33.4824510Z [INFO] Services ........................................... SKIPPED
2021-08-03T22:20:33.4825068Z [INFO] Repository ......................................... SKIPPED
2021-08-03T22:20:33.4825873Z [INFO] Repository - Database .............................. SKIPPED
2021-08-03T22:20:33.4826714Z [INFO] Core - Generation - Mustache ....................... SKIPPED
2021-08-03T22:20:33.4827554Z [INFO] Core - Generation - Velocity ....................... SKIPPED
2021-08-03T22:20:33.4828434Z [INFO] Core - Generation - Javascript ..................... SKIPPED
2021-08-03T22:20:33.4829245Z [INFO] Core - Registry .................................... SKIPPED
2021-08-03T22:20:33.4830016Z [INFO] Core - Migrations .................................. SKIPPED
2021-08-03T22:20:33.4830570Z [INFO] Core ............................................... SKIPPED
2021-08-03T22:20:33.4831105Z [INFO] Database ........................................... SKIPPED
2021-08-03T22:20:33.4831847Z [INFO] Database - Custom .................................. SKIPPED
2021-08-03T22:20:33.4833339Z [INFO] Database - Managed ................................. SKIPPED
2021-08-03T22:20:33.4833944Z [INFO] Commons ............................................ SKIPPED
2021-08-03T22:20:33.4834668Z [INFO] CMS - Managed ...................................... SKIPPED
2021-08-03T22:20:33.4835949Z [INFO] API - Facade - Documents ........................... SKIPPED
2021-08-03T22:20:33.4838006Z [INFO] IDE - UI - Documents ............................... SKIPPED
2021-08-03T22:20:33.4838868Z [INFO] IDE - UI - Documents - Ext Content Type MS ......... SKIPPED
2021-08-03T22:20:33.4839655Z [INFO] IDE - UI - CSV ..................................... SKIPPED
2021-08-03T22:20:33.4840381Z [INFO] IDE - UI - CSVIM ................................... SKIPPED
2021-08-03T22:20:33.4840920Z [INFO] CMS ................................................ SKIPPED
2021-08-03T22:20:33.4842921Z [INFO] CMS - Database ..................................... SKIPPED
2021-08-03T22:20:33.4843833Z [INFO] BPM - Parent ....................................... SKIPPED
2021-08-03T22:20:33.4844556Z [INFO] BPM - API .......................................... SKIPPED
2021-08-03T22:20:33.4845301Z [INFO] BPM - Flowable ..................................... SKIPPED
2021-08-03T22:20:33.4846048Z [INFO] API - Facade - BPM ................................. SKIPPED
2021-08-03T22:20:33.4847345Z [INFO] Resources - Parent ................................. SKIPPED
2021-08-03T22:20:33.4848194Z [INFO] API - JavaScript - BPM ............................. SKIPPED
2021-08-03T22:20:33.4848949Z [INFO] IDE - UI - BPM ..................................... SKIPPED
2021-08-03T22:20:33.4849696Z [INFO] IDE - Service - BPM ................................ SKIPPED
2021-08-03T22:20:33.4850512Z [INFO] Resources - Flowable Libraries ..................... SKIPPED
2021-08-03T22:20:33.4851127Z [INFO] BPM ................................................ SKIPPED
2021-08-03T22:20:33.4851852Z [INFO] Security - Parent .................................. SKIPPED
2021-08-03T22:20:33.4852875Z [INFO] Security - OAuth ................................... SKIPPED
2021-08-03T22:20:33.4853466Z [INFO] Security ........................................... SKIPPED
2021-08-03T22:20:33.4853990Z [INFO] OData .............................................. SKIPPED
2021-08-03T22:20:33.4854732Z [INFO] Group - Runtime - All .............................. SKIPPED
2021-08-03T22:20:33.4855547Z [INFO] Group - Runtime - JavaScript ....................... SKIPPED
2021-08-03T22:20:33.4856383Z [INFO] Releng - Anonymous - All ........................... SKIPPED
2021-08-03T22:20:33.4857197Z [INFO] Releng - Anonymous - Runtime ....................... SKIPPED
2021-08-03T22:20:33.4858016Z [INFO] Releng - Desktop - All ............................. SKIPPED
2021-08-03T22:20:33.4858820Z [INFO] Releng - OpenShift - All ........................... SKIPPED
2021-08-03T22:20:33.4859598Z [INFO] Releng - SAP - All ................................. SKIPPED
2021-08-03T22:20:33.4860594Z [INFO] Releng - SAP - All - Ephemeral ..................... SKIPPED
2021-08-03T22:20:33.4861440Z [INFO] Releng - SAP - CF - Base ........................... SKIPPED
2021-08-03T22:20:33.4862214Z [INFO] Releng - SAP - CF - All ............................ SKIPPED
2021-08-03T22:20:33.4862992Z [INFO] Releng - SAP - CF - Runtime ........................ SKIPPED
2021-08-03T22:20:33.4863794Z [INFO] Releng - SAP - Kyma - Base ......................... SKIPPED
2021-08-03T22:20:33.4864578Z [INFO] Releng - SAP - Kyma - All .......................... SKIPPED
2021-08-03T22:20:33.4865382Z [INFO] Releng - SAP - Kyma - Runtime ...................... SKIPPED
2021-08-03T22:20:33.4866170Z [INFO] Releng - SAP - CMS ................................. SKIPPED
2021-08-03T22:20:33.4866975Z [INFO] Releng - SAP - Runtime - Ephemeral ................. SKIPPED
2021-08-03T22:20:33.4867811Z [INFO] Releng - Server - All .............................. SKIPPED
2021-08-03T22:20:33.4868612Z [INFO] Releng - Server - Keycloak - All ................... SKIPPED
2021-08-03T22:20:33.4869447Z [INFO] Releng - Server - OAuth ............................ SKIPPED
2021-08-03T22:20:33.4870242Z [INFO] Releng - Server - Runtime .......................... SKIPPED
2021-08-03T22:20:33.4871088Z [INFO] Releng - Server - Runtime - Keycloak ............... SKIPPED
2021-08-03T22:20:33.4871916Z [INFO] Releng - Trial - All ............................... SKIPPED
2021-08-03T22:20:33.4872880Z [INFO] Releng - Server - Minimal .......................... SKIPPED
2021-08-03T22:20:33.4873679Z [INFO] Releng - Parent .................................... SKIPPED
2021-08-03T22:20:33.4874517Z [INFO] ------------------------------------------------------------------------
2021-08-03T22:20:33.4875115Z [INFO] BUILD FAILURE
2021-08-03T22:20:33.4875883Z [INFO] ------------------------------------------------------------------------
2021-08-03T22:20:33.4876488Z [INFO] Total time:  08:07 min
2021-08-03T22:20:33.4877132Z [INFO] Finished at: 2021-08-03T22:20:33Z
2021-08-03T22:20:33.4877931Z [INFO] ------------------------------------------------------------------------
2021-08-03T22:20:33.4879665Z [ERROR] Failed to execute goal org.apache.maven.plugins:maven-surefire-plugin:2.22.2:test (default-test) on project dirigible-api-facade-rabbitmq: There are test failures.
2021-08-03T22:20:33.4880927Z [ERROR] 
2021-08-03T22:20:33.4882127Z [ERROR] Please refer to /home/runner/work/dirigible/dirigible/ext/ext-api/api-rabbitmq/target/surefire-reports for the individual test results.
2021-08-03T22:20:33.4883607Z [ERROR] Please refer to dump files (if any exist) [date].dump, [date]-jvmRun[N].dump and [date].dumpstream.
2021-08-03T22:20:33.4884443Z [ERROR] -> [Help 1]
2021-08-03T22:20:33.4884834Z [ERROR] 
2021-08-03T22:20:33.4885622Z [ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
2021-08-03T22:20:33.4886623Z [ERROR] Re-run Maven using the -X switch to enable full debug logging.
2021-08-03T22:20:33.4887178Z [ERROR] 
2021-08-03T22:20:33.4888049Z [ERROR] For more information about the errors and possible solutions, please read the following articles:
2021-08-03T22:20:33.4889242Z [ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
2021-08-03T22:20:33.4890047Z [ERROR] 
2021-08-03T22:20:33.4890673Z [ERROR] After correcting the problems, you can resume the build with the command
2021-08-03T22:20:33.4891789Z [ERROR]   mvn <args> -rf :dirigible-api-facade-rabbitmq
2021-08-03T22:20:33.5056264Z ##[error]Process completed with exit code 1.
```

--- a/ext/ext-api/api-rabbitmq/src/test/java/org/eclipse/dirigible/api/rabbitmq/RabbitMQFacadeTest.java
+++ b/ext/ext-api/api-rabbitmq/src/test/java/org/eclipse/dirigible/api/rabbitmq/RabbitMQFacadeTest.java
@@ -30,7 +30,7 @@ public class RabbitMQFacadeTest {
 	
 	@Before
 	public void setUp() {
-		RabbitMQContainer rabbit = new RabbitMQContainer("rabbitmq:alpine");
+		RabbitMQContainer rabbit = new RabbitMQContainer("rabbitmq:3.8.19-alpine");
 		rabbit.start();
 		
 		String host = rabbit.getHost();

